guides/micronaut-data-jdbc-repository/java/src/main/java/example/micronaut/domain/Book.java

```diff
@@ -0,0 +1,80 @@
+package example.micronaut.domain;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.ManyToOne;
+import javax.persistence.Table;
+import javax.validation.constraints.NotNull;
+
+@Entity
+@Table(name = "book")
```

Comment:
Why the use of javax annotations? Seems unnecessary

Reply:
This guide is a port of [Access a database with mybatis](https://guides.micronaut.io/latest/micronaut-data-access-mybatis.html) and [access a database with JPA and Hibernate](https://guides.micronaut.io/latest/micronaut-jpa-hibernate.html).  I think it is a good example of how Micronaut Data JDBC simplifies development since this guide basically deletes code because the repository is implemented by Micronaut Data. 

`Genre`, the entity class, is identical in those guides and in this Micronaut Data JDBC guide. I think it is valuable to show that you can use the JPA annotations you are used to.

Reply:
The guides are meant to show how we believe developers should write applications. I don't think it is the case that we recommend developers use JPA annotations. It is just another dependency you have to add which is unnecessary. If we want to call it out by pointing to the Micronaut Data documentation that seems reasonable,  but I don't see why we should be recommending JPA annotations by default. In particular it just forces the developer to add another dependency. If it was our recommending when you created an app with Micronaut Data JDBC in Launch the JPA dependency should be there as well which it is not.

Reply:
I see this the same way as Sergio. I think developers are used to use the JPA annotations and learn ours annotations is an additional extra step that some of them probably don't want to do. Specially when the JPA annotations work and they have better support in IDEs.

Reply:
Ok so in that case why aren't we creating the project in launch with the JPA annotations included?

Reply:
Also I must point out that the point of Micronaut Data JDBC is not to accommodate what developers are used to. The example in fact is using mutable entities and getters/setters

Micronaut Data JDBC's design is to encourage the use of immutable entities and mapping JDBC results to entities. It is not an ORM tool but a simple data mapper, therefore we should be showing patterns that demonstrate how it is designed to be used, not just porting a JPA application and using the same patterns.

Reply:
In addition if we are going to use the JPA annotations we will need to point out in the guide that with JDBC only a subset of the JPA annotations are supported and that since Data JDBC is not a full ORM developers cannot expect that every annotation works. We should point to a reference documentation which describes which annotations are in fact supported otherwise this will lead to confusion for readers of the guide.